src/shared/schema/index.js

```diff
@@ -286,6 +286,13 @@ export const WalletSettingsSchema = {
             type: 'bool',
             default: false,
         },
+        /**
+         * Determines if deep linking is enabled
+         */
+        deepLinking: {
```

Comment:
@laumair do we need to bump the `schemaVersion` for this change?

Reply:
Yeah, and we will need to write a migration. @laumair is gonna refactor migrations as a whole, then we can add this one.